oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/measure/BanyanDBMetadataQueryDAO.java

```diff
@@ -201,7 +201,7 @@ protected void apply(MeasureQuery query) {
     }
 
     @Override
-    public List<Process> listProcesses(String serviceId, String instanceId, String agentId, ProfilingSupportStatus profilingSupportStatus, long lastPingStartTimeBucket, long lastPingEndTimeBucket) throws IOException {
+    public List<Process> listProcesses(String serviceId, String instanceId, String agentId, ProfilingSupportStatus profilingSupportStatus, boolean excludeVirtual, long lastPingStartTimeBucket, long lastPingEndTimeBucket) throws IOException {
```

Comment:
In any case, would `boolean excludeVirtual` be false?

Reply:
`excludeVirtual` should be true to exclude all the virtual processes. Do you mean we don't need this parameter, just hard code to exclude all virtual processes in the query?

Reply:
Yes, I mean do we need in any case in the future, we would set this to `false`?

Reply:
No scenario for now. The network topology query should query the processes through ID only.

Reply:
Then, let's remove this parameter until we need it. OK?

Reply:
Sure